assets/queries/k8s/default_service_account/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "default_service_account",
+  "queryName": "Default Service Account",
```

Comment:
Query name should identify the problem with the resource, in this case, that default service accounts are being used.